### PR TITLE
DEV: add outlet wrapper for small user list

### PR DESCRIPTION
**Pull request by branquinhoaa**
- This PR adds one extra argument to an existing outlet
- Adds another outlet for users like list `small-user-list-internal`
- Move the function `smallUserAtts` to a service so it can be extensible and avoids code duplication

### CODE REVIEW

File: app/assets/javascripts/discourse/app/services/user-list-atts.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Service from "@ember/service";
+import { userPath } from "discourse/lib/url";
+
+export default class UserListAtts extends Service {
+  smallUserAtts(user) {
```
[the remark this answers is not in the document]
Comment (branquinhoaa) [reply app/assets/javascripts/discourse/app/services/user-list-atts.js:5]: Hey @CvX I took a look at this and tried to refactor as you asked.

This function is generic and works for multiple types of user models (user, post action user, post reader) so I would need to duplicate the logic. Apart from that, some of these models are not explicit defined (ie:post action user). 

Let me know if you still think it is better to move or if you have any other idea.

[the remark this answers is not in the document]
Comment (CvX) [reply app/assets/javascripts/discourse/app/services/user-list-atts.js:5]: Ok, so let's move it to `lib` directory. Services are a better fit for stuff with state and/or multiple related methods (used by various components, etc) but here we have just a single-input utility function.

also, can you rename `atts` to `attrs`? 😅 that contraction is way more common in core

[the remark this answers is not in the document]
Comment (branquinhoaa) [reply app/assets/javascripts/discourse/app/services/user-list-atts.js]: @CvX as we discussed in chat, I replaced the implementation to use Value Transformers and it works like a charm! thank you for the tip 👯

**Review by CvX — APPROVED** [review]

File: app/assets/javascripts/discourse/app/components/small-user-list.gjs [hunk 2]
```diff
@@ -1,31 +1,23 @@
 import Component from "@glimmer/component";
+import { hash } from "@ember/helper";
 import { service } from "@ember/service";
+import PluginOutlet from "discourse/components/plugin-outlet";
 import avatar from "discourse/helpers/bound-avatar-template";
-import { userPath } from "discourse/lib/url";
+import { smallUserAttrs } from "discourse/lib/user-list-attrs";
+import i18n from "discourse-common/helpers/i18n";
```
Comment (CvX) [inline app/assets/javascripts/discourse/app/components/small-user-list.gjs:7]: ah, imports need a fixup 😅 
```suggestion
```

**Pull request merged**